guide/popular-topics/threads.md

```diff
@@ -0,0 +1,153 @@
+- `Client#threadDelete`: Emitted whenever a thread is deleted.
+- `Client#threadUpdate`: Emitted whenever a thread is updated - e.g. name change, archive state change, locked state change.
+- `Client#threadListSync`: Emitted whenever the client user gains access to a text or news channel that contains threads.
+- `Client#threadMembersUpdate`: Emitted whenever members are added or removed from a thread. Requires `GUILD_MEMBERS` privileged intent.
+- `Client#threadMemberUpdate`: Emitted whenever the client user's thread member is updated.
+
+## Creating and deleting threads
+
+Threads are created and deleted using the `ThreadManager` of a text or news channel.
+To create a thread you call the `ThreadManager#create()` method:
+
+```js {2-8}
+client.on('messageCreate', async message => {
+	const thread = await message.channel.threads
+		.create({
+			name: 'food-talk',
+			autoArchiveDuration: 60,
+			reason: 'Needed a separate thread for food',
+		});
+	console.log(`Created thread: ${thread.name}`);
+});
+```
+
+To delete a thread you call the `ThreadChannel#delete()` method:
+
+```js {2,3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.delete();
+});
+```
+
+## Joining and leaving threads
+
+To join your Client to a ThreadChannel you make use of the `ThreadChannel#join()` method:
+
+```js {3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.join();
+});
+```
+
+And to leave one you call `ThreadChannel#leave()`;
+
+```js {3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.leave();
+});
+```
+
+## Archiving, unarchiving and locking threads
+
+Every thread can be either active or archived. Changing a thread from archived to active is referred to as unarchiving the thread. Threads that have `locked` set to true can only be unarchived by a member with the `MANAGE_THREADS` permission.
+
+Threads automatically archive after inactivity. 'Activity' is defined as sending a message, unarchiving a thread, or changing the auto-archive time.
+
+To archive a thread you use the `ThreadChannel#setArchived()` method and pass `true` as parameter:
+
+```js {3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.setArchived(true);
+});
+```
+
+And to unarchive it one you pass `false ` to one and the same method:
+
+```js {3}
+client.on('messageCreate', async message => {
+	const thread = message.channel.threads.cache.find(x => x.name === 'food-talk');
+	await thread.setArchived(false);
+});
+```
+
+This same principle applies to locking and unlocking a thread via the `ThreadChannel#setLocked()` method:
```

Comment:
Probably some note about not being able to lock when the thread is archived would be useful